js/utils.js

```diff
@@ -48,11 +48,28 @@ export function setattr(obj, name, value) {
     obj[name] = value;
 }
 
+/**
+ * Parse a query string into an object
+ * @param  {String} qs  A querystring, with or without the leading '?'
+ * @return {Object}     The parsed query string
+ */
+export function parseQS(qs) {
```

Comment:
BabelJS doesn't know about browser, DOM or anything related to this, it only transpile ES6 to ES5. If we want to use this API, we have to find a polyfill (which might be very close from this snippet and potentially bigger).
I can't wait for browser to have native support for the whole ES6 stack and the new DOM/HTML5 APIs !!